students/barecki_andrzej/lesson_07_string_datetime/date_calculator.py

```diff
@@ -0,0 +1,26 @@
+import datetime
+
+
+def main():
+    now = datetime.datetime.now()
+    print("Current date and time: {0:04}-{1:02}-{2:02} {3:02}:{4:02}:{5:02}"
+          .format(now.year, now.month, now.day,
+                  now.hour, now.minute, now.second))
```

Comment:
Datetime udostępnia wygodniejsze metody to formatowania czasu.

[Zerknij jak to zrobił inny z kursantów](https://github.com/jedzej/tietopythontraining-basic/blob/6b34b6b70059e21bb0d9a5df336e2abc8c3df270/students/semko_krzysztof/lesson_07_strings_manipulation/date_calculator.py)